Dockerfile

```diff
@@ -1,7 +1,8 @@
+#syntax=docker/dockerfile:1.7-labs
 FROM eclipse-temurin:17 AS build
 
 WORKDIR /app
-COPY . .
+COPY --exclude=entrypoint.sh --exclude=docker-compose.yaml --exclude=Jenkinsfile --exclude=README.md --exclude=mock-ldap --exclude=.github --exclude=.git . .
```

Comment:
I don't think there's much point excluding all these other paths, 

dockerignore can do the generic stuff.
Then the specific ignore that you want here is the entrypoint one so it doesn't rebuild on entrypoint change?

Reply:
True, I guess I over complicated things. Fixed with your recommendation, thanks for the reviw